### ISSUE TITLE
Unable to add instantsearch-core-swift via swiftpm

### ISSUE THREAD

**Issue by rhettdickson** [algolia/instantsearch-core-swift#89]
**Describe the bug 🐛**
Unable to add this package to swift package manager. Receive this error on attempt:

```
the package instantsearch-core-swift[https://github.com/algolia/instantsearch-core-swift.git] @ 6.3.2 contains incompatible dependencies:
    algoliasearch-client-swift[https://github.com/algolia/algoliasearch-client-swift] @ release/7.0.2
```

**To Reproduce 🔍**
1. Go to your own Xcode project's settings
2. Click on Swift Packages
3. Click plus button
4. Enter `https://github.com/algolia/instantsearch-core-swift.git` for package repository URL

**Expected behavior 💭**
Package resolution succeeds

**Screenshots 🖥**
<img width="730" alt="image" src="https://user-images.githubusercontent.com/720958/71856316-66700100-3098-11ea-9346-bd5bf0ba2f88.png">

**Environment:**
 - OS: macOS Catalina

**Comment by spinach** [comment]
Hi @rhettdickson,

Thank you for reporting this, and sorry for the late reply! 

This ongoing PR #88 will solve your issue. If you want, you can target `feat/insights` branch in order for it to build. We'll release a new version that has the fix in the coming days.

**Comment by VladislavFitz** [comment]
Resolved in [version 6.4](https://github.com/algolia/instantsearch-core-swift/releases/tag/6.4.0)
